release-scripts/src/main/java/software/amazon/awssdk/release/CreateNewServiceModuleMain.java

```diff
@@ -48,18 +55,46 @@
  * </pre>
  */
 public class CreateNewServiceModuleMain extends Cli {
+
+    private static final Set<String> DEFAULT_INTERNAL_DEPENDENCIES = toSet("http-auth-aws");
```

Comment:
Can we add some Javadoc to mention this?

Reply:
done.